lib/carrier/credentials.ex

```diff
@@ -1,19 +1,14 @@
 defmodule Carrier.Credentials do
 
-  defstruct [:id, :private, :public, :tag]
+  defstruct [:id, :tag]
 
   @cred_db "carrier_credentials.db"
-  @private_key "carrier_priv.key"
-  @public_key "carrier_pub.key"
   @carrier_id "carrier.id"
 
-  # 32 byte key w/64 byte checksum
-  @key_hash_size 64
-
   @doc "Generates a new private/public keypair"
```

Comment:
@christophermaier It will eventually go away, but we still need it because we use the uid generated here to identify relays. I'm thinking that we can replace the entire module with something more generic like `Carrier.Registry` or something. Something that we could use to store addresses to relays. I didn't change it because I wanted to have a better idea and get some more input first.